pkgs/development/tools/selenium/selendroid/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
     makeWrapper ${jdk}/bin/java $out/bin/selendroid-selenium \
       --add-flags "-Dfile.encoding=UTF-8" \
       --add-flags "-cp \"$out/share/lib/selendroid/${name}.jar:${selenium-server-standalone}/share/lib/${selenium-server-standalone.name}/${selenium-server-standalone.name}.jar\"" \
-      --add-flags "org.openqa.grid.selenium.GridLauncher"
+      --add-flags "org.openqa.grid.selenium.GridLauncherV3"
```

Comment:
I also suspect it needs to be updated: http://selendroid.io/ (websites states it is compatible with v3, but probably only in the latest version).
It would be great if you could do some basic testing (see if it can connect to the selenium server)

Reply:
How would a basic test look like (I've got no experience with selendroid). Maybe checking if devices can register like in the [getting started docs](http://selendroid.io/setup.html#launchingSelendroid)?

Reply:
I also have never heard of selendroid before this pull request. I you can reach http://localhost:4444/wd/hub/status after starting this should be enough. I mainly want to make sure it can properly use the new selenium version.

Reply:
Allright I tested it and it seems to startup fine with current `androidsdk` from nixpkgs. I didn't test to start an actual browser though, just startup and the status endpoint.